src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/sentencing/SentencingResourceIntTest.kt

```diff
@@ -3807,64 +3826,20 @@ class SentencingResourceIntTest : IntegrationTestBase() {
           .jsonPath("offenderCharges[1].offenceDate").isEqualTo(aLaterDateString)
       }
 
-      @Test
-      fun `can create a sentence without a court order`() {
-        // update appearance to have one inactive charge (removing the court order)
```

Comment:
I was kind of expecting new tests for the new exceptions that are thrown - or maybe something like this test to be converted to a negative test?